File: src/main/java/com/uwetrottmann/trakt5/entities/PlaybackResponse.java
package com.uwetrottmann.trakt5.entities;

import org.threeten.bp.OffsetDateTime;

public class PlaybackResponse extends GenericProgress {

    public Long id;
    /** Playpack response only. */
    public OffsetDateTime paused_at;
    /** Playpack response only. */
    public String type;
    /** Scrobble response only. */
    public String action;
    /** Scrobble response only. */
    public ShareSettings sharing;

}

File: src/main/kotlin/com/redissi/trakt/entities/PlaybackResponse.kt
package com.redissi.trakt.entities

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import java.time.OffsetDateTime

@JsonClass(generateAdapter = true)
class PlaybackResponse(
    override val episode: SyncEpisode? = null,
    override val show: SyncShow? = null,
    override val movie: SyncMovie? = null,
    override val progress: Double? = null,
    val id: Long? = null,
    /** Playpack response only.  */
    @Json(name = "paused_at")
    val pausedAt: OffsetDateTime? = null,
    /** Playpack response only.  */
    val type: String? = null,
    /** Scrobble response only.  */
    val action: String? = null,
    /** Scrobble response only.  */
    val sharing: ShareSettings? = null
) : GenericProgress() {

}